hw/dv/tools/dvsim/sim.mk

```diff
@@ -70,26 +67,40 @@ ifneq (${sw_images},)
 	# test> followed by <index>. The <flag> is considered optional.
 	set -e; \
 	mkdir -p ${sw_build_dir}; \
-	${LOCK_SW_BUILD_DIR} "cd ${proj_root} && \
-		env BUILD_ROOT=${sw_build_dir} ${proj_root}/meson_init.sh"; \
 	for sw_image in ${sw_images}; do \
-		image=`echo $$sw_image | cut -d: -f 1`;  \
+		image=`echo $$sw_image | cut -d: -f 1`; \
 		index=`echo $$sw_image | cut -d: -f 2`; \
 		flags=(`echo $$sw_image | cut -d: -f 3- --output-delimiter " "`); \
+		target_dir=`dirname ${sw_build_dir}/build-bin/$$image`; \
+		mkdir -p $$target_dir; \
+		cd ${proj_root}; \
```

Comment:
TODO: someday we'll get rid of `build-bin` in favor of everything being built by bazel and living in `bazel-out/...`

Reply:
I can add a todo here, and file a supporting issue. Preferably in a follow since this PR is merge-conflict prone due to the changes to the chip_sim_cfg.hjson file.